src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AdministrationController.php

```diff
@@ -184,4 +187,71 @@ protected function getNotificationsFormHandler(): FormHandlerInterface
     {
         return $this->get('prestashop.adapter.administration.notifications.form_handler');
     }
+
+    /**
+     * @var FormError[] $errors
+     */
+    private function getErrorMessages(array $errors): array
+    {
+        $messages = [];
+
+        foreach ($errors as $error) {
+            $messages[] = $this->getErrorMessage($error);
+        }
+
+        return $messages;
+    }
+
+    /**
+     * @param FormError $error
+     *
+     * @return string
+     */
+    private function getErrorMessage(FormError $error): string
+    {
+        switch ($error->getErrorCode()) {
+            case UploadQuotaDataProvider::ERROR_NOT_NUMERIC_OR_LOWER_THEN_0:
+                return $this->trans(
+                    '"%s is invalid. Please enter an integer greater or equal to 0."',
```

Comment:
```suggestion
                    '%s is invalid. Please enter an integer greater or equal to 0.',
```

Reply:
I'm not sure if you added the "" on purpose?

Reply:
Yeah I think it was incorrect, thanks!